changelog_a70q.txt

```diff
@@ -0,0 +1,12 @@
+Highlights & Device Specific Changes:
```

Comment:
Bad filename, must be "a70q_changelog.txt"

Reply:
that's what I got in the generated build, also 13.x was the same https://github.com/crdroidandroid/android_vendor_crDroidOTA/blob/13.0/changelog_a70q.txt unles it changes in 14.x, thanks for notify!

Reply:
@tw0xa I updated shall I close the PR and add new one?

Reply:
No need to open a new pr, just correct filename and push :)

Reply:
@tw0xa done :)